scheduler/kubernetes/Logmanagement-fluentd and S3.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: page
+title: "PX Log Storage and Management with S3 using Fluentd"
+keywords: portworx, container, Kubernetes, storage, Docker, S3, k8s, pv, persistent disk, fluentd, cluster logging, log management
+sidebar: home_sidebar
+---
+
+* TOC
+{:toc}
+
+PX Logs management on Kubernetes using Fluentd. 
+
+## PX Logs on Kubernetes
+PX runs as a deamonset on the Kubernetes cluster which ensures that it runs on each node as part of the Kubernetes cluster. To allow access to the logs of a failed node, pod or a container in kubernetes we would have to adopt a complete logging solution. The need to access or view logs of failed container workloads means that we would need to enable storage and the logs should have a seperate lifecycle than that of the container that creates it. 
+
+## Log Collection
+Fluentd is a log collector which enables you to log everything in a unified manner. Fluentd uses JSON as the log collection format. 
+
+
+### Install fluentd on your kubernetes cluster.
+The following instructions allows you to ship your Portworx logs to an S3 bucket managed by Portworx. 
+
+Write to support@portworx.com requesting for an S3 bucket to enable remote storage of the PX cluster logs.
```

Comment:
Instead of just Write to support@portworx.com, also add slack as an option. We have our slack URL and thumbnail on the home page https://docs.portworx.com/